src/platforms/macos/daemon/macosdnsmanager.h

```diff
@@ -0,0 +1,40 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#ifndef MACOSDNSMANAGER_H
+#define MACOSDNSMANAGER_H
+
+#include <systemconfiguration/scdynamicstore.h>
+
+#include <QMap>
+
+#include "command.h"
+
+class MacOSDnsManager final : public Command {
```

Comment:
Also, the naming is a bit confusing because it is a command and a DnsManager. Should those be separated (separation of concerns), by introducing a DnsManagerCommand which will handle the command line args and create the MacOSDnsManager?

Reply:
As for the naming, that was following the precedent set by the `MacOSDaemonServer` which also breaks from our typical `CommandFooBar` class naming convention.